workitem/link/link_repository_blackbox_test.go

```diff
@@ -125,13 +127,92 @@ func (s *linkRepoBlackBoxTest) TestDisallowMultipleParents() {
 
 	// create a work item link
 	linkRepository := link.NewWorkItemLinkRepository(s.DB)
-	_, err = linkRepository.Create(s.ctx, Parent1ID, ChildID, s.testTreeLinkTypeID, s.testIdentity.ID)
+	linkTest, err := linkRepository.Create(s.ctx, Parent1ID, ChildID, s.testTreeLinkTypeID, s.testIdentity.ID)
+	require.Nil(s.T(), err)
+
+	var exists bool
+	exists, err = linkRepository.Exists(s.ctx, linkTest.ID.String())
 	require.Nil(s.T(), err)
+	require.True(s.T(), exists)
 
 	_, err = linkRepository.Create(s.ctx, Parent2ID, ChildID, s.testTreeLinkTypeID, s.testIdentity.ID)
 	require.NotNil(s.T(), err)
 }
 
+func (s *linkRepoBlackBoxTest) TestExistsLink() {
```

Comment:
Why not test this with already [existing link types](https://github.com/almighty/almighty-core/blob/master/workitem/link/type.go#L21) and all the other resources that you need? The you probably only have to create two work items and link them.